app/assets/src/components/PipelineSampleReads.jsx

```diff
@@ -102,6 +102,27 @@ class PipelineSampleReads extends React.Component {
     );
   }
 
+  render_metadata_textfield_wide(label, field) {
+    return (
+      <div className="col s12">
+        <div className="details-title note">{label}</div>
+        <div className="sample-notes note">
+          <pre
+            className="details-value"
+            suppressContentEditableWarning={true}
```

Comment:
Great PR 🎉 ! 

The only thing I can see is for prop `suppressContentEditableWarning={true}` we could just omit the `={true}` part and do `suppressContentEditableWarning`, per https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md.

E.g.,
```tsx
// No good
const Hello = <Hello personal={true} />;

// Good
const Hello = <Hello personal />;
```